main.py

```diff
@@ -2,6 +2,8 @@
 
 def main():
     print('Hello, World!')
```

Comment:
Let's not make this such a strong statement.

```suggestion
    print('Hello, World?')
```